addon/components/validated-input/template.hbs

```diff
@@ -31,7 +35,19 @@
         {{option.label}}</label>
     </div>
   {{/each}}
-
+{{else if (eq type "checkbox")}}
+  <div class="{{config.css.checkbox}}">
+    <label>
+      {{one-way-checkbox
+          checked
+          name       = name
+          value      = value
```

Comment:
I think this would be too constricting. What if I have a list of checkboxes for say `categories`, which is an array. and I do `{{#each}}` on the array, and each of these items has a `selected` or `checked` property. In this case the input would be unusable - how do you get a boolean value based on array?. So I think it is best to leave it to the end developer to put  `checked = (get model name)`  in the `checked` property, and provide examples of use with a single checkbox, and checkboxes as a group